corrector.rb

```diff
@@ -0,0 +1,5 @@
+class Corrector
```

Comment:
Kindly ensure to provide a _Base Decorator_ which:

- [ ] Check if the `Decorator` class inherits from Namable.
- [ ] Check if in the constructor a namable object from params is assigned to an instance variable.
- [ ] Check of the correct_name method that returns the result of the correct_name method of the @namable is implemented.

The decorator base should look like the following:
```ruby
class Decorator < Nameable
    attr_accessor :nameable
  
    def initialize(nameable)
      @nameable = nameable
    end
  

    def correct_name
      @nameable.correct_name
    end
end
```